src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/PhysicalProjectTree.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    [Export(typeof(IPhysicalProjectTree))]
+    internal class PhysicalProjectTree : IPhysicalProjectTree
```

Comment:
I wonder if it's worth using this in the SpecialFileProviders to see if that simplifies the code there.